tilavarauspalvelu/tasks.py

```diff
@@ -602,6 +606,15 @@ def generate_reservation_series_from_allocations(application_round_id: int) -> N
         create_or_update_reservation_statistics.delay(reservation_pks=list(reservation_pks))
 
 
+@app.task(name="delete_expired_applications")
+def delete_expired_applications() -> None:
+    Application.objects.filter(
+        L(status__in=[ApplicationStatusChoice.EXPIRED, ApplicationStatusChoice.CANCELLED])
+        & L(application_round__status=ApplicationRoundStatusChoice.RESULTS_SENT)
+        & Q(application_round__application_period_end__lte=local_date() - datetime.timedelta(days=365))
```

Comment:
Should this time be configurable?

Reply:
Could be useful at least for testing purpose, e.g. `REMOVE_EXPIRED_APPLICATIONS_OLDER_THAN_DAYS`

Reply:
Moved the magic value behind an env variable